Importer/Log/CsvLog.php

```diff
@@ -50,9 +50,12 @@ public function addDataError($index, $errorMessage)
     /**
      * {@inheritdoc}
      */
-    public function addCriticalError($errorMessage)
+    public function addCriticalError($message, $userError = false)
```

Comment:
Ok. But at the moment in u2, we also log the `errorReason` when it is caused by a crush. Which means that for a manual application error, we will show to user a readable error message, but we will also log the **errorReason**. 

And as @yakobe said, we do not want to log the abort messages. So this approach does not apply to the demands of u2.